<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Шифратор</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layoutBttns">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Сменить язык:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLanguage">
       <property name="text">
        <string>Русский</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnColumn">
       <property name="text">
        <string>Зашифровать столбцовым методом</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnVigenere">
       <property name="text">
        <string>Зашифровать методом Виженера (прогр. ключ)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStoreData">
       <property name="text">
        <string>Загрузить исходный текст из файла</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDecrypt">
       <property name="text">
        <string>Расшифровать текущее</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDecryptFromFile">
       <property name="text">
        <string>Расшифровать данные из файла</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>401</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layoutPlaintext">
     <item>
      <widget class="QLabel" name="labelPlaintext">
       <property name="text">
        <string>Исходный текст:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="editPlaintext"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>411</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layoutED">
     <item>
      <layout class="QVBoxLayout" name="layoutD">
       <item>
        <widget class="QLabel" name="labelDecryption">
         <property name="text">
          <string>Шифротекст:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="editDecryption">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutE">
       <item>
        <widget class="QLabel" name="labelEncryption">
         <property name="text">
          <string>Расшифрованный текст:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="editEncryption">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layoutKey">
     <item>
      <widget class="QLabel" name="labelKey">
       <property name="text">
        <string>Ключ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editKey"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
